ui/apps/akasha-verse/src/components/pages/explore-page.tsx

```diff
@@ -8,12 +8,9 @@ import AppList from '@akashaorg/design-system-components/lib/components/AppList'
 import Text from '@akashaorg/design-system-core/lib/components/Text';
 import { useTranslation } from 'react-i18next';
 import { APP_EVENTS } from '@akashaorg/typings/sdk';
-import { RootComponentProps } from '@akashaorg/typings/ui';
 import { GetAppsQuery, GetAppsByIdQuery } from '@akashaorg/typings/sdk/graphql-operation-types-new';
-import { IntegrationReleaseInfo } from '@akashaorg/typings/sdk/graphql-types';
-import { INFO } from '../../routes';
 
-export interface IExplorePage extends RootComponentProps {
+export interface IExplorePage {
```

Comment:
convert to type

Reply:
Alright. There are lots of other places where interface was used. Converting all might mean greater number of files being changed. Do I do it on this same PR?

Reply:
we'll do it in steps, whenever you change or update one, also change it to type